editor/src/dev/yourkit.clj

```diff
@@ -222,7 +222,9 @@
 ;; Allocation recording
 ;; -----------------------------------------------------------------------------
 
-(def ^AllocationProfilingSettings default-allocation-profiling-settings (AllocationProfilingSettings.))
+(def ^AllocationProfilingSettings default-allocation-profiling-settings
+  (doto (AllocationProfilingSettings.)
+    (.setMode AllocationProfilingMode/HEAP_SAMPLING)))
```

Comment:
Changed the allocation recording mode to `HEAP_SAMPLING`, which has way less overhead.